.env

```diff
@@ -0,0 +1,16 @@
+#Enviroment NGINX
+NGINX_TAG=1.13
+
+#Enviroment PHP
+PHP_TAG=7.1-fpm
+
+#Enviroment MYSQL
+MYSQL_TAG=5.6.35
+
+#Enviroment Apache
+APACHE_TAG=2.4
+
+#Enviroment GLPI
+CLONNING_METHOD=ssh
+GLPI_BRANCH=9.2/bugfixes
+GLPI_SOURCE=git@github.com:glpi-project/glpi.git
```

Comment:
Hi, @ingluife 
Maybe a new env var, for the server name: `dev.glpi.local`

Reply:
Good, i will add it as environment variable.